### ISSUE TITLE
[Bug]: (CRITICAL) Invulnerable Wither Storm rapidly inflicts Wither Sickness with no counter

### ISSUE THREAD

**Issue by GearsawStudios** [nonamecrackers2/crackers-wither-storm-mod#1922]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Minecraft Version

1.20.1

### Mod Version

4.1

### Mod Loader

Forge

### Mod Loader Version

47.2.0

### Describe the bug

A bug happened!

Being near the Wither Storm as it spawns will rapidly inflict Wither Sickness. My personal hypothesis is that it inflicts incredibly high amounts of contacts while spawning in.

### Steps to Reproduce

1. Summon the Wither Storm in survival in the intended manner (Optionally, decrease the Application Delay to 12 to see the effects faster without skewing the results.)
2. Run
3. See Wither Sickness

### Crash Reports/Logs

Note: I don't think these are of any use...

### Is OptiFine installed?

- [ ] OptiFine is installed

### Screenhots

https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/122060314/96899241-dde2-4d91-85ee-1bafd831dc2b

### Mod List

CWSM
JEI

**Comment by GearsawStudios** [comment]
[message (2).txt](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/15313150/message.2.txt)
Logs, pressed enter to make a new line and it posted instead.

**Comment by ConnorTG5** [comment]
i can confirm that i also had this same issue
it being a problem with the wither's invulnerable state specifically makes sense since i dont remember having this problem when i played on versions before the large lag spike that followed the wither spawning in was fixed, and this lag spike may have allowed people to get far enough away to avoid being instantly infected.

**Comment by ConnorTG5** [comment]
i was doing hardcore runs and decided to just use cheats to get around this bug but i noticed something odd
this bug doesn't seem to happen when the world has cheats enabled, it might just be a coincidence but the pattern remained consistent across about 8 attempts

**Comment by Cod1ngGamer** [comment]
Maybe this was already patched in a way that used cheats somehow?

**Comment by ConnorTG5** [comment]
update: it might be LAN and cheats at the same time that prevents it

**Comment by GearsawStudios** [comment]
New info on this, turns out this applies to all phases and means that getting close to the WS at all immediately starts application regardless of contacts or proximity time.

**Comment by sm644me** [comment]
Found out more info about this from my own testing. It looks like the "contacts" system of getting Wither Sickness is broken. I went into Survival mode during a regular phase 4 chase, and in less than 30 seconds, I had over 200 contacts despite a beam never touching me. This is definitely not supposed to happen, but I won't make a new bug report for it due to it probably being the same thing as mentioned here.

EDIT: On 1.19.4, if that matters.
